### PR TITLE
Address derivation and portability

### PR DESCRIPTION
**Pull request by joepetrowski**
Keys, keys, keys, keys;
Makin' crypto keys.

Supersedes https://github.com/w3f/polkadot-wiki/pull/449

### CODE REVIEW

File: docs/learn-accounts.md [hunk 1]
```diff
@@ -26,8 +26,78 @@ These prefixes are defined
 
 It's important to understand that the different formats for different networks are merely different
 representations of the same public key in a private-public keypair generated by an address
-generation tool (see next section). This makes the addresses cross-Substrate-chain-compatible as
-long as you convert the format.
+generation tool. This makes the addresses cross-Substrate-chain-compatible as long as you convert
```
Comment (lsaether) [inline docs/learn-accounts.md]:
```suggestion
generation tool. This makes the addresses compatible across Substrate-based chains as long as you convert
```

File: docs/learn-accounts.md [hunk 2]
```diff
@@ -26,8 +26,78 @@ These prefixes are defined
...
+phrase in multiple wallets. Wallets that use different steps will arrive at a different set of
+addresses from the same mnemonic.
+
+### Seed Generation
+
+Most wallets generate a mnemonic phrase for users to back up their wallets and generate a private
+key from the mnemonic. Not all wallets use the same algorithm to convert from mnemonic phrase to
+private key.
+
+Subkey and Polkadot JS based wallets use the BIP39 dictionary for mnemonic generation, but use the
```
Comment (lsaether) [inline docs/learn-accounts.md]:
```suggestion
Subkey and Polkadot-JS based wallets use the BIP39 dictionary for mnemonic generation, but use the
```

File: docs/learn-accounts.md [hunk 3]
```diff
@@ -26,8 +26,78 @@ These prefixes are defined
...
+entropy byte array to generate the private key, while full BIP39 wallets (like Ledger) use 2048
+rounds of PBKDF2 on the mnemonic. As such, the same mnemonic will not generate the same private
+keys. See [Substrate BIP39](https://github.com/paritytech/substrate-bip39) for more information.
+
+### Derivation Paths
+
+Many Polkadot key generation tools support hard and soft derivation. See the
+[Subkey documentation](https://substrate.dev/docs/en/knowledgebase/integrate/subkey) for details and
+examples of derivation path formats. The Polkadot JS Apps and Extension as well as Parity Signer
```
Comment (lsaether) [inline docs/learn-accounts.md]:
```suggestion
examples of derivation path formats. The Polkadot-JS Apps and Extension as well as Parity Signer
```

File: docs/learn-accounts.md [hunk 4]
```diff
@@ -26,8 +26,78 @@ These prefixes are defined
...
+- Seed derivation
+- Signature scheme
+
+If you want to use the same mnemonic across multiple wallets, make sure that they follow compatible
+methods for generating keys and signing messages. If you cannot find understandable documentation,
+reach out to the project maintainers.
+
+|                       | Mnemonic Format | Derivation Path | Seed Derivation |      Signature Support      |
+| :-------------------- | :-------------: | :-------------: | :-------------: | :-------------------------: |
+| Polkadot JS Extension |    Standard     |  User-Defined   |      BIP32      |           sr25519           |
```
Comment (lsaether) [inline docs/learn-accounts.md]:
```suggestion
| Polkadot-JS Extension |    Standard     |  User-Defined   |      BIP32      |           sr25519           |
```

File: docs/learn-accounts.md [hunk 5]
```diff
@@ -26,8 +26,78 @@ These prefixes are defined
...
+| Polkadot JS Apps      |   Standard\*    |  User-Defined   |      BIP32      | sr25519, ed25519, secp256k  |
```
Comment (lsaether) [inline docs/learn-accounts.md]:
```suggestion
| Polkadot-JS Apps      |   Standard\*    |  User-Defined   |      BIP32      | sr25519, ed25519, secp256k  |
```

**Review by lsaether — COMMENTED** [review]
No major changes requested, just typographical stuff.

**Review by Swader — APPROVED** [review]

**Review by lsaether — APPROVED** [review]

**Pull request merged**